scrapegraphai/utils/copy.py

```diff
@@ -0,0 +1,78 @@
+import copy
+from typing import Any, Dict, Optional
+
+
+def safe_deepcopy(obj: Any, memo: Optional[Dict[int, Any]] = None) -> Any:
```

Comment:
why `memo` and not a more standard `cache`?

Reply:
I deleted it because I found it not very necessary and may cause other problems. BTW,memo is the second parameter name of deepcopy so I use it.